dali/operators/image/crop/bbox_crop.cc

```diff
@@ -354,7 +354,16 @@ The values are:
         R"code(If set to True, an extra output will be returned, containing
 the original indices of the bounding boxes that passed the centroid filter and are present in
 the output bounding boxes.)code",
-        false);
+        false)
+    .AddOptionalArg<float>("bbox_prune_threshold",
+        R"code(Controls when bboxes are considered outside of the ROI and pruned. If this argument is set,
+boxes are kept if the fraction of their area within the ROI is greater than or equal to the threshold specified
+`[0.0,1.0]`. If this argument is **not** set, boxes are pruned if their centroid is outside of the ROI.
+
+For example, when `bbox_prune_threshold=0.2` bboxes that have more than 20% of their original area within
+the ROI are kept, bboxes less than or equal to are pruned. If `bbox_prune_threshold=0.0`, all boxes that
+have some presence in the ROI are kept.)code",
+        0.0f);
```

Comment:
```suggestion
have some presence in the ROI are kept.)code");
```
There's no need to set a default value if you are going to read it only when provided (the documentation will be clearer too)